src/ClockMock.php

```diff
@@ -12,14 +15,19 @@
  */
 final class ClockMock
 {
-    private static bool $areMocksActive = false;
+    private static ?DateTimeInterface $frozenDateTime = null;
 
-    private static ?\DateTimeInterface $frozenDateTime = null;
+    private static ?DateTimeInterface $originalRequestDateTime = null;
+
+    private static string $mode = self::MODE_REALTIME;
+
+    private const MODE_REALTIME = 'REALTIME';
+    private const MODE_FROZEN   = 'FROZEN';
```

Comment:
I have goals of adding a time travel functionality, replacing the boolean mocksActive with a generic mode will facilitate that. This changes does not affect existing functionality